ui/ui-components/pages/model/[modelId]/records.tsx

```diff
@@ -1,7 +1,10 @@
+import { NextPageContext } from "next";
 import Head from "next/head";
 import RecordsList from "../../../components/record/List";
 
-export default function Page(props) {
+export default function Page(
+  props: Awaited<ReturnType<typeof RecordsList.hydrate>>
```

Comment:
Should probably be typeof `Page.getInitialProps` instead. I wonder if this warrants a quick utility in `pageHelper` that does this such as `InferGetInitialPropsType`